201-api-management-logs-to-moesif-using-eventhub-webapp/nested/microsoft.web/sites.json

```diff
@@ -0,0 +1,200 @@
+                "hyperV": false,
+                "siteConfig": {
+                    "appSettings": [
+                        {
+                            "name": "APIMEVENTS-EVENTHUB-CONNECTIONSTRING",
+                            "value": "[listKeys(resourceId('Microsoft.EventHub/namespaces/authorizationRules', parameters('eventHubNamespace'), parameters('eventHubListenPolicy')), '2017-04-01').primaryConnectionString]"
+                        },
+                        {
+                            "name": "APIMEVENTS-EVENTHUB-NAME",
+                            "value": "[parameters('eventHubName')]"
+                        },
+                        {
+                            "name": "APIMEVENTS-STORAGEACCOUNT-NAME",
+                            "value": "[parameters('apimEvtStorName')]"
+                        },
+                        {
+                            "name": "APIMEVENTS-STORAGEACCOUNT-KEY",
+                            "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', parameters('apimEvtStorName')), '2019-06-01').keys[0].value]"
+                        },
+                        {
+                            "name": "APIMEVENTS-MOESIF-APPLICATION-ID",
+                            "value": "[parameters('apimEvtMoesifApplicationId')]"
+                        },
+                        {
+                            "name": "APIMEVENTS-MOESIF-SESSION-TOKEN",
+                            "value": "[parameters('apimEvtMoesifSessionToken')]"
+                        },
+                        {
+                            "name": "APIMEVENTS-MOESIF-API-VERSION",
+                            "value": "[parameters('apimEvtMoesifApiVersion')]"
+                        }
+                    ]
+                },
+                "scmSiteAlsoStopped": false,
+                "clientAffinityEnabled": true,
+                "clientCertEnabled": false,
+                "hostNamesDisabled": false,
+                "containerSize": 0,
+                "dailyMemoryTimeQuota": 0,
+                "httpsOnly": false,
+                "redundancyMode": "None"
+            }
+        },
+        {
+            "type": "Microsoft.Web/sites/config",
+            "apiVersion": "2018-11-01",
+            "name": "[concat(parameters('appServiceName'), '/web')]",
+            "location": "[parameters('location')]",
+            "tags": "[parameters('tags')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/sites', parameters('appServiceName'))]",
+                "[resourceId('Microsoft.Web/sites/hostNameBindings', parameters('appServiceName'), variables('appServiceHostName'))]"
+            ],
+            "properties": {
+                "netFrameworkVersion": "v5.0",
+                "logsDirectorySizeLimit": 60,
+                "numberOfWorkers": 1,
+                "alwaysOn": "true", // Not supported for F1 and D1 plan. It will fail deployment.
```

Comment:
if the sample supports F1/D1 then make this a parameter and note the exception in the description.  If not supported use allowedValues on the sku

Reply:
will update shortly - in azuredeploy and this comment as well